packages/webapp/pages/posts/[id]/comments/[commentId]/edit.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import CommentInputPage from '@dailydotdev/shared/src/components/comments/CommentInputPage';
+
+export interface Props {
+  id: string;
+  commentId: string;
+}
+
+const EditCommentPage = ({ id, commentId }: Props): ReactElement => {
```

Comment:
edit comment page renders the same component as new comment page, just with different props